spec/index.html

```diff
@@ -805,23 +805,25 @@ <h3>Reified Triples</h3>
     <p>A <a>reified triple</a> is represented using the
       <a href="#grammar-production-reifiedTriple"><code>reifiedTriple</code></a> production
       starting with <a href="#cp-double-lt"><code>&lt;&lt;</code></a>,
-      followed by <a href="#grammar-production-subject"><code>subject</code></a>,
+      followed by <a href="#grammar-production-ttSubject"><code>ttSubject</code></a>,
       <a href="#grammar-production-predicate"><code>predicate</code></a>, and
       <a href="#grammar-production-ttObject"><code>ttObject</code></a>,
-      optionally follwed by a <a href="#grammar-production-reifier"><code>reifier</code></a>
-      (<a href="#grammar-production-IRIREF"><code>IRIREF</code></a>
-      or <a href="#grammar-production-BlankNode"><code>BlankNode</code></a>),
+      optionally followed by a <a href="#grammar-production-reifier"><code>reifier</code></a>,
+      composed of a <a href="#cp-tilda"><code title="tilda">~</code></a>
+      followed by an <a href="#grammar-production-iri"><code>iri</code></a>
+      or <a href="#grammar-production-BlankNode"><code>BlankNode</code></a>,
       and ending with <a href="#cp-double-gt"><code>&gt;&gt;</code></a>.
       For example, `&lt;&lt; :subject :predicate :object ~ :IRIREF &gt;&gt;`.
-      If the optional reifier is not present, a fresh RDF blank node is allocated,
+      If the optional reifier is not present,
+      a fresh <a data-cite="RDF12-CONCEPTS#dfn-blank-node">RDF blank nodes</a> is allocated,
       as with `&lt;&lt; :subject :predicate :object &gt;&gt;`.</p>
 
     <p class="note"><a>Reified Triples</a> may be nested,
-      like `&lt;&lt;&nbsp;&nbsp;:subject1&nbsp;:predicate1&nbsp;&lt;&lt;&nbsp;:subject2&nbsp;:predicate2&nbsp;:object2&nbsp;&gt;&gt;&nbsp;~:IRIREF1&nbsp;&gt;&gt;`
-      or `&lt;&lt;&nbsp;:subject4&nbsp;:predicate4&nbsp;&lt;&lt;&nbsp;&nbsp;:subject3&nbsp;:predicate3&nbsp;:object3&nbsp;~:IRIREF3&nbsp;&gt;&gt;&nbsp;&gt;&gt;`.</p>
+      like `&lt;&lt;&nbsp;:subject1&nbsp;:predicate1&nbsp;&lt;&lt;&nbsp;:subject2&nbsp;:predicate2&nbsp;:object2&nbsp;&gt;&gt;&nbsp;~:IRIREF1&nbsp;&gt;&gt;`
+      or `&lt;&lt;&nbsp;:subject4&nbsp;:predicate4&nbsp;&lt;&lt;&nbsp;:subject3&nbsp;:predicate3&nbsp;:object3&nbsp;~:IRIREF3&nbsp;&gt;&gt;&nbsp;&gt;&gt;`.</p>
 
     <p>If a <a href="#grammar-production-reifiedTriple"><code>reifiedTriple</code></a>
-      is not identified by an <a href="#grammar-production-IRIREF"><code>IRIREF</code></a>
+      is not identified by an <a href="#grammar-production-iri"><code>iri</code></a>
```

Comment:
```suggestion
      is not identified by an <a href="#grammar-production-iri">IRI</a>
```

Reply:
The terminal is `iri`, not `IRI`. There is an `IRIREF` (used previously), which is the strict IRI grammar, and does not allow prefixed names. This change allows prefixed names.

Reply:
The difference I see is reference to "`iri`" and possibly also "`BlankNode`" (which I think should be "IRI" and possibly "BlankNode" in text, though they be linked to `#grammar-production-iri` and `#grammar-production-BlankNode`), as here in —
```
... not identified by an <a href="#grammar-production-iri"><code>iri</code></a> 
or <a href="#grammar-production-BlankNode"><code>BlankNode</code></a> ...
```
— vs "`iri` production" and "`BlankNode` production" (which I can understand being "`iri`" in text, also linked to `#grammar-production-iri`), as in —
```
... an identifier from either the <a href="#grammar-production-iri"><code>iri</code></a>
or <a href="#grammar-production-BlankNode"><code>BlankNode</code></a> productions ...
```

I think that if you want to keep written as "`iri`" the instances that I suggested be changed to "IRI", those segments should be edited to something like —
```
... not identified by an identifier from either the  
<a href="#grammar-production-iri"><code>iri</code></a> 
or the <a href="#grammar-production-BlankNode"><code>BlankNode</code></a> 
production ...
```

Reply:
Good point about not using the `iri` production when what is intended is the `IRI` concept. Same for `BlankNode`.